core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -88,16 +88,16 @@
 import io.fabric8.openshift.api.model.Route;
 import io.fabric8.openshift.api.model.RouteSpecBuilder;
 import io.fabric8.openshift.api.model.Template;
-import io.fabric8.openshift.client.DefaultOpenShiftClient;
 import io.fabric8.openshift.client.OpenShiftConfig;
 import io.fabric8.openshift.client.OpenShiftConfigBuilder;
 import io.fabric8.openshift.client.ParameterValue;
+import io.fabric8.openshift.client.impl.OpenShiftClientImpl;
 import lombok.extern.slf4j.Slf4j;
 import rx.Observable;
 import rx.observables.StringObservable;
 
 @Slf4j
-public class OpenShift extends DefaultOpenShiftClient {
+public class OpenShift extends OpenShiftClientImpl {
```

Comment:
Are we sure we want to extend on an `*Impl` class?

Reply:
You are right, I've noticed the issue when I tried to use 6.7.2 version of the client as Mirek suggested. I'm going to have to come up with something else that hopefuly doesnt break all the other changes.

Reply:
> .. Went for replacing inheritance with composition 

Sounds good to me.